.travis.yml

```diff
@@ -9,3 +9,18 @@ matrix:
           - docker build -t bubblesub .
           script:
           - docker run -ti bubblesub
+        - os: windows
+          services: docker
+          before_deploy:
+          - travis_wait 50 docker build -t bubblesub -f windows-build/Dockerfile .
+          - windows-build/deploy-binary.bat
+          deploy:
+              provider: releases
+              api_key:
+                secure: qOm6rK82TsBfIKxpn8hTTnTWPsJl0g/WrzXcnoiwiulFZtoxq0QzJUz5iYNZYfJbVvHhZEea9QLBWTNwyHK50QGdMTMXvDHaxq942p59DajnpQyRdmEmyzu34TqP7W49jZCm41yH4+yPVE9wkokGUaXGUDCKk321LmHwHAqJ4oh243L2W5SgvAmMbR8ZuTmeLw8FUD8E209OhxI8CQTNHAlaA6uWSs+KhrFYsrgVWUB8liZ/Z0W2OkzeDzsw8enFKL1MBvgv+aa3f4ICgjqeNmSl1IB8o6bJP5Ndzq2KTZUY22ei0wP/es/uLDRnWkD+313CkEE5oaPvOPddDuE3VWC8+ed/CBlUo2178BnTDNhAd+E9W9xAkUV0YcKoydO30L047HlYMwdnh7FnT8lx5fqhCY9HIvBxrwglA8RDLHpD0M/SeJMLaMphcoYnad1kxv3TpFHIf7N6LHlIqvYmLXUo1ENYKACrzYjrwpMV+DYFTK8wac3UdPHXxKRaxTCMRhA09/kJyfgHxxwBEAPCP91+3W6dtvFWKf/UvMli0aMCovT4RSmT20PUcLYYNWA+77wh3mxjuVhQxeamb1QtwK3XK+o0+03kTVASNH7gJdAoaje9+rhr9vxe+K88IA/AqB+52w/1SIz7LrNLed/lum9khf97IdcF1Ns0JjmxtzM=
```

Comment:
Is it a private key, or a public key? Is it okay for it to be public? (I think it is, but words like `api key` and `secure` triggers red flags)

Reply:
It is a private key.  It represents my Github token encrypted. I have used the following command to get it:

`travis encrypt --com  GithubTokens`

More information about this topic [here](https://docs.travis-ci.com/user/encryption-keys/)

Reply:
Okay, it's a private key encrypted with a public key that is readable only on travis machines.